andela/dark-knight-cfh
# #156881766  Integrate houndci

#### What does this PR do?
Setup Houndci
#### Description of Task to be completed?
1. Create .hound.yml file
2. Write settings for houndci in .yml file

#### How should this be manually tested?
1. Create a new pull request
2. Verify that hound checks for eslint errors without error

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#156881766 

#### Screenshots (if appropriate)
N/A
